src/mixins/meta.ts

```diff
@@ -127,3 +161,44 @@ export function metaTag(
 
   return metaTagsObject;
 }
+
+/**
+ * Alternate locales meta and link tags generator.
+ * @param locales Array of alternate locales the the webpage is currently available in.
+ * @param pageCurrentLocale
+ * @returns 
+ */
+function pageAlternateLocales(locales: QMetaI18nAlternateLocale[], pageCurrentLocale?: QMetaI18nAlternateLocale) {
+  const currentDomain = domain();
+  
+  const linksAlternate: {[index: string]: AlternateLocaleLink } = {};
+  const metaAlternate: {[index: string]: OpenGraphMetaTag} = {};
+
+  for (const {locale, url} of locales) {
+    linksAlternate["alt-" + locale] = {
+      href: currentDomain + url,
+      hreflang: locale,
+      rel: "alternate",
+    };
+
+    // [1] - The open graph locale only wants the language used as the value for the content property. See Open Graph https://ogp.me/
+    metaAlternate["og-alt-" + locale] = {
+      content: locale,
+      property: "og:locale:alternate",
+    };
+  }
+
+  // [1]
+  // In addition, Open Graph also wants to know the locale of the current page in this way.
+  if (pageCurrentLocale) {
```

Comment:
`pageCurrentLocale` should not be provided manually, but reactively update based on `$i18n.locale` (see the watch in this mixin), selecting the correct one from provided `locales` array.
If there isn't a match into locales, current i18n language and current route will be used to fill in the property